### DISCUSSION TITLE
Object analytics help

### DISCUSSION THREAD

**Discussion by qxdevelop** [AxisCommunications/discussions#117] [VAPIX]
Hi, I need to collect info about the actual vehicle occupation on the scene. I'm wondering about using metadata for collecting this info is the right way for obtaining this. I do not find any other way for getting this information using Object analytics API. Can you help me with this?

**Comment by tpatora** [comment]
https://pandosme.github.io/acap/2022/06/03/parking-transition.html

**Comment by vivekatoffice** [comment]
Good to explore [metadata-broker](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/metadata-broker) example
